go.mod

```diff
@@ -2,7 +2,17 @@ module github.com/openshift/windows-machine-config-operator
 
 go 1.12
 
+// Replace is used to pin a specific version of a package or to point to sub-go.mod directories.
+// Use 'replace' to point to the sub-go.mod directory for building a binary in the root directory and always build by
+// package instead of file.
+replace github.com/openshift/windows-machine-config-operator/tools/windows-node-installer => ./tools/windows-node-installer
```

Comment:
Not a huge fan of this. Why would we need `tools/windows-node-installer` package if it's not being imported in @Bowenislandsong please address this in a separate PR